servers/physics_3d/area_3d_sw.cpp

```diff
@@ -163,6 +163,20 @@ void Area3DSW::set_param(PhysicsServer3D::AreaParameter p_param, const Variant &
 		case PhysicsServer3D::AREA_PARAM_PRIORITY:
 			priority = p_value;
 			break;
+		case PhysicsServer3D::AREA_PARAM_WIND_FORCE_MAGNITUDE:
+			wind_force_magnitude = p_value;
+			ERR_FAIL_COND_MSG(wind_force_magnitude >= 0, "Wind force magnitude must be a non-negative real number, but a negative number was specified.");
+			break;
+		case PhysicsServer3D::AREA_PARAM_WIND_SOURCE:
+			wind_source = p_value;
+			break;
+		case PhysicsServer3D::AREA_PARAM_WIND_DIRECTION:
+			wind_direction = p_value;
+			break;
+		case PhysicsServer3D::AREA_PARAM_WIND_ATTENUATION_FACTOR:
+			wind_attenuation_factor = p_value;
+			ERR_FAIL_COND_MSG(wind_force_magnitude >= 0, "Wind attenuation factor must be a non-negative real number, but a negative number was specified.");
```

Comment:
Same as the previous one, and the condition should check `wind_attenuation_factor ` instead of `wind_force_magnitude`.

Reply:
Same, the error should be before setting the value.

Reply:
I was looking at the wrong line in my code. Got it.